diff --git a/node_modules/react-native-pell-rich-editor/src/editor.js b/node_modules/react-native-pell-rich-editor/src/editor.js
index c51f4bf..27f6892 100644
--- a/node_modules/react-native-pell-rich-editor/src/editor.js
+++ b/node_modules/react-native-pell-rich-editor/src/editor.js
@@ -565,7 +565,7 @@ function createHTML(options = {}) {
                     if ( action[msgData.name]){
                         var flag = msgData.name === 'result';
                         // insert image or link need current focus
-                        flag && focusCurrent();
+                        // flag && focusCurrent();
                         action[msgData.name](msgData.data, msgData.options);
                         flag && handleState();
                     } else {
